### ISSUE TITLE
label for pretrain

### ISSUE THREAD

**Issue by liu-x-p** [andi611/Self-Supervised-Speech-Pretraining-and-Representation-Learning#52]
Hi, sorry to bother again.
I want to use part of the data with my own label during pretraining, so how to train the upstream model supervised?
Could I train it in supervised way firstly with my label, and then use the model to the downstream task with the cpc_phone label? I'm afraid the two different sets of phone labels would confuse the model.

**Comment by andi611** [comment]
Hi,

I'm not sure what you meant by "label during pretraining", pre-training DO NOT require any label.
If you want to train it supervised, you need to implement it yourself.
This repo focuses on self-supervised pre-training, hence we will not implement supervised pre-training.

> Could I train it in supervised way firstly with my label, and then use the model to the downstream task with the cpc_phone label? I'm afraid the two different sets of phone labels would confuse the model.

I'm not sure what the goal is, so it is hard for me to answer this question.
I guess you can, but I don't know if it will work.

**Comment by liu-x-p** [comment]
I think I made some mistakes in the question. I want to use the pre-training module to train the model with labels. Then I will try to modify the code.
Thank you!

**Comment by andi611** [comment]
I see!
